Kea is annoying with its opinionated alteration of certain paths.
So we revert what it does.

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -59,7 +59,7 @@ SYSCONFDIR = get_option('sysconfdir')
 
 # Meson is annoying with its opinionated alteration of certain paths based on whether prefix is default or not.
 # So we revert what it does.
-if PREFIX == '/usr/local'
+if false and PREFIX == '/usr/local'
     LOCALSTATEDIR = 'var'    # Otherwise, it would have been 'var/local'.
 endif
 
@@ -80,18 +80,18 @@ else
     RUNSTATEDIR = RUNSTATEDIR / 'kea'
 endif
 RUNSTATEDIR_INSTALLED = PREFIX / RUNSTATEDIR
-SHAREDSTATEDIR = LOCALSTATEDIR / 'lib/kea'
+SHAREDSTATEDIR = LOCALSTATEDIR / 'db/kea'
 SHAREDSTATEDIR_INSTALLED = PREFIX / SHAREDSTATEDIR
 SYSCONFDIR_INSTALLED = PREFIX / SYSCONFDIR
 TEST_CA_DIR = TOP_SOURCE_DIR / 'src/lib/asiolink/testutils/ca'
 
 # Meson is annoying with its opinionated alteration of certain paths based on whether prefix is default or not.
 # So we revert what it does.
-if PREFIX == '/usr/local'
+if false and PREFIX == '/usr/local'
     SYSCONFDIR_INSTALLED = '/usr/local/etc'
     LOGDIR_INSTALLED = '/usr/local/var/log/kea'
     RUNSTATEDIR_INSTALLED = '/usr/local/var/run/kea'
-    SHAREDSTATEDIR_INSTALLED = '/usr/local/var/lib/kea'
+    SHAREDSTATEDIR_INSTALLED = '/usr/local/var/db/kea'
 endif
 
 #### Build Options
@@ -1146,9 +1146,9 @@ install_emptydir(SHAREDSTATEDIR, install_mode: 'rwxr-x
 # Meson is annoying with its opinionated alteration of certain paths based on whether prefix is default or not.
 # So we revert what it does..
 # In case prefix is default, install to hardcoded path.
-if PREFIX == '/usr/local'
+if false and PREFIX == '/usr/local'
     install_emptydir('/usr/local/lib/kea', install_mode: 'rwxr-x---')
-    install_emptydir('/usr/local/var/lib/kea', install_mode: 'rwxr-x---')
+    install_emptydir('/usr/local/var/db/kea', install_mode: 'rwxr-x---')
     install_emptydir('/usr/local/var/log/kea', install_mode: 'rwxr-x---')
     install_emptydir('/usr/local/var/run/kea', install_mode: 'rwxr-x---')
 endif
